packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -144,6 +148,22 @@ const isWithinAvailableHours = (
       return true;
     }
   }
+
+  // check if it is a date override
+  for (const dateOverride of dateOverrides) {
+    const utcOffSet = dayjs(dateOverride.start).tz(inviteeTimeZone).utcOffset();
+
+    const slotStart = dayjs(timeSlotStart).add(utcOffSet, "minute");
+    const slotEnd = dayjs(timeSlotEnd).add(utcOffSet, "minute");
+
+    if (
+      slotStart.isBetween(dateOverride.start, dateOverride.end) &&
```

Comment:
We are having a bigger known issue with date overrides which is that date overrides currently are not adjusted to the timezone. 

So if the schedule has a date override from 1PM - 3PM, then no matter what timezone the booking person has it will always show the 1PM - 3PM slots (in their timezone) open. This is something we need to fix separately (it's quite complex)

For now, when you log the date override and the slot start it should both be the time of how the slots are shown for the booking person. 

@hariombalhara let me know if that makes sense to you and if you have any other concerns with that implementation 🙏